<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geo Json Viewer</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Geo Json Viewer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="point_button">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="GeoJsonViewer.qrc">
          <normaloff>:/Resources/icons8-select-90.png</normaloff>:/Resources/icons8-select-90.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="line_string_button">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="GeoJsonViewer.qrc">
          <normaloff>:/Resources/icons8-polyline-90.png</normaloff>:/Resources/icons8-polyline-90.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="polygon_button">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="GeoJsonViewer.qrc">
          <normaloff>:/Resources/icons8-polygon-64.png</normaloff>:/Resources/icons8-polygon-64.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="save_button">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="GeoJsonViewer.qrc">
          <normaloff>:/Resources/icons8-back-100.png</normaloff>:/Resources/icons8-back-100.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="exit_button">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="GeoJsonViewer.qrc">
          <normaloff>:/Resources/icons8-forward-100.png</normaloff>:/Resources/icons8-forward-100.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="open_button">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="GeoJsonViewer.qrc">
          <normaloff>:/Resources/icons8-open-sign-100.png</normaloff>:/Resources/icons8-open-sign-100.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="fit_in_view_button">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="GeoJsonViewer.qrc">
          <normaloff>:/Resources/icons8-thumbnail-view2.png</normaloff>:/Resources/icons8-thumbnail-view2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
      <item>
       <widget class="RasterGraphicsView" name="graphics_view"/>
      </item>
      <item>
       <widget class="QTableView" name="table_view">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RasterGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">rastergraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="GeoJsonViewer.qrc"/>
 </resources>
 <connections/>
</ui>
